pkg/infoschema/issyncer/loader.go

```diff
@@ -64,11 +96,14 @@ type Loader struct {
 
 // NewLoaderForCrossKS creates a new Loader instance.
 func NewLoaderForCrossKS(store kv.Storage, infoCache *infoschema.InfoCache) *Loader {
+	mode := LoadModeFull
```

Comment:
I found `(is *infoschemaV2) keepAlive()` encountered the same issue previously. It is possible to trigger TS update periodically instead of switching to v1 completely?

Reply:
the loaded infoschema is using v1, `keepAlive()` is a v2 method which will not be called